File: RagnarokBotWeb/Domain/Services/Dto/OrderDto.cs
using Shared.Enums;

namespace RagnarokBotWeb.Domain.Services.Dto
{
    public class OrderDto
    {
        public long Id { get; set; }
        public PackDto? Pack { get; set; }
        public WarzoneDto? Warzone { get; set; }
        public TaxiDto? Taxi { get; set; }
        public EOrderType OrderType { get; set; }
        public EOrderStatus Status { get; set; }
        public PlayerDto? Player { get; set; }
        public ScumServerDto ScumServer { get; set; }
        public DateTime CreateDate { get; set; }
    }
}

File: RagnarokBotWeb/ClientApp/src/app/models/order.dto.ts
import { PackageDto } from "./package.dto";
import { PlayerDto } from "./player.dto";
import { ScumServer } from "./scum-server";
import { TaxiDto } from "./taxi.dto";
import { WarzoneDto } from "./warzone.dto";

export class OrderDto {
    id!: number;
    pack?: PackageDto;
    warzone?: WarzoneDto;
    taxi?: TaxiDto;
    status: number;
    orderType: number;
    player?: PlayerDto;
    scumServer!: ScumServer;
    createDate: Date;
}

export interface GraphDto {
    name: string;
    amount: number;
    color: string;
}